judge/judgedaemon.main.php

```diff
@@ -543,14 +543,14 @@ function fetch_executable_internal(
 
 // Set static environment variables for passing path configuration
 // to called programs:
-putenv('DJ_BINDIR='      . BINDIR);
-putenv('DJ_ETCDIR='      . ETCDIR);
-putenv('DJ_JUDGEDIR='    . JUDGEDIR);
-putenv('DJ_LIBDIR='      . LIBDIR);
+putenv('DJ_BINDIR=' . BINDIR);
+putenv('DJ_ETCDIR=' . ETCDIR);
+putenv('DJ_JUDGEDIR=' . JUDGEDIR);
+putenv('DJ_LIBDIR=' . LIBDIR);
```

Comment:
I don't like this removal of alignment spaces. This specific place is maybe not the best example of where it is useful, but there are better places where I feel strongly that alignment improves code readability a lot and should not be sacrificed.

Reply:
Yeah let's disable this rule.